### DISCUSSION TITLE
请问，在 nuxt 生成的页面中，如何将组件的样式自动生成到一个文件中呢

### DISCUSSION THREAD

**Discussion by lip8up** [tusen-ai/naive-ui#3253] [Q&A]
通过这个 [plugins](https://github.com/07akioni/naive-ui-nuxt-demo/blob/main/plugins/naive-ui.ts)，配合 nuxt3 项目，生成的页面，将组件样式放在每个页面的 <head> 中，以内联 <style ssr-id="n-button"></style> 的形式。

直接放在 head 中，一是有点乱，二是不同的页面大量重复。

谢谢！

**Comment by 07akioni** [comment]
这个应该是做不到的。

我理解某次 SSG 之后后面应该就是 CSR 接管了，按理来说不应该收到所谓的重复样式的影响。
